### PR TITLE
removing yrb parser in favor of intl-messageformat pkg

### PR DESCRIPTION
**Pull request by caridy**
- remove transpiler in favor for `require('intl-messageformat').__parse`
- parsed messages are now arrays

_THIS IS BLOCK UNTIL INTL-MESSAGEFORMAT GETS RELEASED_
## TODO
- [x] docs
- [x] tests
- [x] examples

### CODE REVIEW

**Comment by caridy** [comment]
this is not a blocker for flickr.

File: lib/plugin.js [hunk 1]
```diff
@@ -86,12 +86,13 @@ PluginClass.prototype = {
...
                     transpiler = require('./transpilers/' + transpiler);
                     for (token in langEntries) {
-                        if (langEntries.hasOwnProperty(token)) {
+                        if (langEntries.hasOwnProperty(token) && typeof langEntries[token] === 'string') {
                             langEntries[token] = transpiler(langEntries[token]);
                         }
                     }
                 }
             } catch (e) {
+                console.log(e, e.stack);
```
Comment (imalberto) [inline lib/plugin.js]: Might be better to use `console.error`.

**Comment by imalberto** [comment]
+1

**Comment by ericf** [comment]
For the record, I don't like this :) `require('intl-messageformat').__parse`

**Comment by caridy** [comment]
- locator-lang@0.1.2

**Comment by ekashida** [comment]
+1

**Pull request merged**